src/components/DataMessenger/DataMessenger.js

```diff
@@ -421,9 +442,8 @@ export default class DataMessenger extends React.Component {
       }),
     ]
 
-    if (_get(this.props.queryQuickStartTopics, 'length')) {
+    if (this.state.topics.length > 0) {
```

Comment:
May want to turn this into _get(this.state, 'topics.length') just in case something goes wrong with the api call?